api/envoy/extensions/filters/http/header_to_metadata/v3/header_to_metadata.proto

```diff
@@ -129,4 +129,7 @@ message Config {
 
   // The list of rules to apply to responses.
   repeated Rule response_rules = 2;
+
+  // Configure max_header_value_len.
```

Comment:
This comment isn't very useful.  What does this value actually do?  And what is the behavior if the header is longer than this size?

Reply:
```
Configure maximum lenth for :ref:`header value 
<envoy_v3_api_field_extensions.filters.http.header_to_metadata.v3.Config.KeyValuePair.value>` 
to be added in metadata. If this value exceed, no metadata will be added. If not specified, default is 8KiB.
```

Reply:
Is this make sense?

Reply:
That sounds fine.  Please update the comment in the PR.

Reply:
sure, thanks